pkg/kube/client.go

```diff
@@ -669,12 +670,21 @@ func WaitForCacheSync(stop <-chan struct{}, cacheSyncs ...cache.InformerSynced)
 		}
 		return true
 	}
+	attempt := 0
+	defer func() {
+		if r {
+			log.WithLabels("name", name, "attempt", attempt, "time", time.Since(t0)).Debugf("sync complete")
```

Comment:
Not sure if you remove Debug() from the API - but if using the structured mode I don't think Printf should be used. Not a huge deal if we move later to upstream slog.

Reply:
Also - we may want to discuss defining some constants for the labels so we have more consistency ( not only for the structured logs, but also for metrics and istiod-generated traces when we add them).